yarn-error.log

```diff
@@ -0,0 +1,78 @@
+Arguments: 
```

Comment:
This file should be in the `.gitignore`

Reply:
Cool, added it to the `.gitignore` file.